### ISSUE TITLE
focus on <main> when a new page renders

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#74]
When a new page renders, the focus should shift to the `<main>` element for accessibility.

Add a test for this to the app shell tests, if possible.

**Comment by vadekh** [comment]
@dwhieb after some initial research, it seems that the `<main>` element is not focusable. I could be wrong. We can override that, but it makes me wonder if we should just pick a different element that is within the standard for focusing.

**Comment by dwhieb** [comment]
@vadekh Huh, go figure. I suppose what we'll have to do then is whenever something renders as the result of a user-initiated action, we'll need to consider whether we should shift the focus to what's been rendered. For example, when rendering the Languages page for a specific language, we'll want to shift the focus to the first input element (which would be the title of the language). There are times we shouldn't do that too though. It'll depend on whether it makes sense for the user to be taken to the newly-rendered element.
